Update pack documentation to state it uses last tagged commit (#973)
StackStorm/st2docs: docs/source/packs.rst

    # Install your own pack from git
    st2 pack install https://github.com/emedvedev/chatops_tutorial

> By default, the latest tagged commit to a pack repository will be installed, but you can specify a particular

Reviewer: _Tagged commit_ looks a bit weird, how about this?

```suggestion
By default, the latest release from the pack repository will be installed, but you can specify a particular
```
Any better examples?
